script/results.html

```diff
@@ -175,81 +530,365 @@
+      text2 += '<div class="accordion" id="results-accordion">'
+      // First we will have a left column in the table with the summarized results
+      text3 += '<tr><td class="align-top"><b>Total:</b> ' + tests_total + '<br><b>Passed:</b> ' + tests_passed + '<br><b>Skipped:</b> ' + tests_skipped + '<br><b>Failed:</b> ' + tests_failed + '</td><td>'
+      text3 += '<div class="accordion" id="results-accordion">'
+      for (const key in input) {
+        //  console.log("key =",key)
+        for (const key2 in input2) {
+
+          eqData = input2[key2].FarEdge
+
+          if (tableName == "telco") {
+            eqData = input2[key2].Telco
+
+          }
+          if (tableName == "nontelco") {
+            eqData = input2[key2].NonTelco
+          }
+
+          if (key == key2) {
+
+            text = ''
+            // NOTE: we are assuming the test result is determined by the passed/failed state of the first item
+            status = input[key][0].state
+            if (status === "passed") {
+              buttontype = 'bg-success text-white'
+            } else if (status === "skipped") {
+              buttontype = 'bg-warning text-black'
+            } else {
+              buttontype = 'bg-danger text-white'
+            }
+
+            itemid = 'collapse' + id
+            headingid = 'heading' + id
+            text += '<div class="accordion-item"><h2 class="accordion-header" id="' + headingid + '"><button class="accordion-button collapsed ' + buttontype + '" type="button" data-bs-toggle="collapse" data-bs-target="#' + itemid + '" aria-expanded="true" aria-controls="' + itemid + '">'
+            text += key + '</button></h2>'
+            // Now we should populate the item contents
+            text += '<div id="' + itemid + '" class="accordion-collapse collapse" aria-labelledby="' + headingid + '">'
+            text += '<div class="accordion-body">'
+            // Inside the accordion, 1 table with the following colummns
+            text += '<table class="table table-bordered table-fixed"><thead><tr>'
+            text += '<th scope="col" class="th-sm">Test ID</th>'
+            text += '<th scope="col" class="th-sm">Test Text</th>'
+            text += '<th scope="col" class="th-sm">Duration</th>'
+            text += '<th scope="col" class="th-sm">State</th>'
+            text += '<th scope="col" class="th-lg">Test output</th>'
+            text += '</tr></thead><tbody>'
+            // Note we are skipping some columns for now
+            // And 1 row per test
+            for (const item of input[key]) {
+              text += '<tr><td>' + item.testID.id + '</td>'
+              text += '<td>' + item.testText.replace(/\n/g, '<br>') + '</td>'
+              text += '<td>' + item.duration + '</td>'
```

Comment:
Convert duration into human readable format with :           
           dayjs.extend(window.dayjs_plugin_duration);
            for (const item of input[key]) {
              const duration = dayjs.duration(item.duration/1000000)
              formattedDuration = duration.format('D[d] H[h] m[m] s[s] SSS[ms]')
             ...
              text += '<td>' + formattedDuration + '</td>'